src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -31,7 +31,7 @@
       }"
     >
       <!-- List Headers -->
-      <v-row v-if="showHeaders" class="people-roles-list-header list-item__subtitle pb-3" no-gutters>
+      <v-row v-if="showHeaders" class="people-roles-list-header list-item__subtitle pt-0 pb-4" no-gutters>
```

Comment:
Whitespace when it's just the list:

![image](https://user-images.githubusercontent.com/10002889/216398036-28bb939a-62df-4ba9-9f1c-ce7bdf7a7a91.png)

Reply:
Whitespace during editing:

![image](https://user-images.githubusercontent.com/10002889/216398598-9deb2990-1e5f-459c-bb96-2a0303af2a63.png)